.github/pull_request_template.md

```diff
@@ -96,6 +96,13 @@ so getting this right initially will aid successfully having it
 proceed. Mislocated entries and trailing spaces should be avoided.
 -->
 
+<!--
+In your domain block submission, please include a role-based email address (e.g., hostmaster@example.com, noc@example.com) rather than a personal email address (e.g., jane.doe@example.com) under your organization name, so that the PSL project can maintain contact with your organization, independent of personnel changes.
```

Comment:
```suggestion
In your submission, please include a role-based email address (e.g. security@example.com) rather than a personal email address (e.g. jane.doe@example.com) under your organization name, so that we can maintain contact with your organization, independent of personnel changes.
```
Shorten it a bit, it's quite a lot of text for a fairly simple check box, also `security@example.com` would likely be a better example.